src/app/layers/qgsapplayerhandling.cpp

```diff
@@ -136,8 +136,49 @@ void QgsAppLayerHandling::postProcessAddedLayer( QgsMapLayer *layer )
   }
 }
 
-void QgsAppLayerHandling::postProcessAddedLayers( const QList<QgsMapLayer *> & )
+void QgsAppLayerHandling::postProcessAddedLayers( const QList<QgsMapLayer *> &layers )
 {
+  for ( QgsMapLayer *layer : layers )
+  {
+    switch ( layer->type() )
```

Comment:
Is `layer` can be a nullptr?

Reply:
No, those would be filtered out well before it hits this method